elfi/methods/post_processing.py

```diff
@@ -204,8 +204,8 @@ def _adjust(self, i, theta_i, regression_model):
 
     def _input_variables(self, model, sample, summary_names):
         """Regress on the differences to the observed summaries."""
-        observed_summaries = np.stack([model[s].observed for s in summary_names], axis=1)
-        summaries = np.stack([sample.outputs[name] for name in summary_names], axis=1)
+        observed_summaries = np.stack([model[s].observed.ravel() for s in summary_names], axis=1)
+        summaries = np.stack([sample.outputs[name].ravel() for name in summary_names], axis=1)
```

Comment:
I'm a bit worried that this might break something. But I guess this is no longer necessary, since the old 1D Gauss is available again?

Reply:
We can leave this case as it was as we decided to keep the old Gauss model. Though this might be a potential issue when a summary statistics function maps to an output larger than one dimension.

Reply:
Better to fail than to give wrong results.